### ISSUE TITLE
Unexpected result in 100PRINT sample

### ISSUE THREAD

**Issue by Lattay** [aldernero/sketchy#8]
Hi,

On my machine (arch linux, go version go1.21.6 linux/amd64) the 100PRINT produce only straight lines.
Investigating the value of the noise variable in the init function, every value is between 0.76 and 0.81, producing only one type of tiles.

Tweaking the persistence parameter I get the same visual result with values of noise above 1.

It seems that Noise2D doesn't produce valid values.

![bug](https://github.com/aldernero/sketchy/assets/24568527/e2da7cb7-3196-4537-8458-1a23091b8f13)

**Comment by aldernero** [comment]
Thanks for noticing this. I was experimenting with the OpenSimplex generator that sketchy uses, and forgot to undo some of the changes regarding signed vs unsigned noise once I reverted the experiments. I fixed the [bug](https://github.com/aldernero/gaul/commit/9c7ed484e7c85b5093d56004705b2bc75ef2693d) and updated sketchy. I also updated the slider defaults. If you update sketchy it should work now, let me know if it doesn't. Here is an example of what it looks like for me after the the update:
![image](https://github.com/aldernero/sketchy/assets/96601789/152dc8aa-575e-42df-ab8c-cb7a4ef8b509)
